README.md

```diff
@@ -1 +1,40 @@
-"# math-magicians" 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math Magicians
+
+> A 3 page project that presents a math calculator with various
+
```

Comment:
- [ ] Kindly write a good description of the project that covers what this project is all about.